meal-planning/src/components/recipe-card/RecipeCard.js

```diff
@@ -12,7 +12,7 @@ import styles from './RecipeCard.module.scss';
 import PropTypes from 'prop-types';
 import DeleteRecipeDialog from '../delete-recipe-dialog/DeleteRecipeDialog';
 
-const RecipeCard = ({ recipe , removeCard}) => {
+const RecipeCard = ({ recipe , removeCard, handleFavoriteSort , handleUnfavorite}) => {
   const [favorite, setFavorite] = useState(false);
```

Comment:
right now we aren't saving the favorite state anywhere. lets add it to the recipe so we can have the state passed into the recipe card. change the default recipes in DefaultRecipes.js to hold the favorite state, and then refactor code to sort on the favorite property. 

{
    id: 1,
    name: 'Curry',
    description: 'spicy coconut curry',
    ingredients: [
      { id: 1, name: 'potato', quantity: 3 },
      { id: 2, name: 'carrot', quantity: 1 },
      { id: 3, name: 'spice', quantity: 1 }
    ],
    favorite: true
  }